### ISSUE TITLE
Regular weight too light to be regular?

### ISSUE THREAD

**Issue by alarcondgtuc** [playbeing/dinish#1]
In here I'm comparing it to a font I use on my browser to replace fonts in websites that have poor readability or make reading less fluid. Here is a comparison between DINish regular and the font I use to replace website fonts.

https://user-images.githubusercontent.com/78399996/180054168-a9d63450-f06c-4195-9c42-c79ccfdb296c.mp4

**Comment by driehuis** [comment]
Your screenshot appears to have less contrast than the rendering of the same page in my browser (Firefox on Linux). The browser used is unspecified, nor is the font being compared with. If you look at the available fonts on Google Fonts or other sources, you will find a wide variety of interpretations of the 400 Regular weight.

Readability and fluidity are of course affected by myriad other factors, such as color contrast, font size, line length etcetera. You may wish to tweak any of those if you want to use DINish.

If you do not like the Regular weight of DINish I'm afraid you will have to look elsewhere for a font to use. There are many DIN styled fonts with differing licenses and conditions, and there may be one that suits you. Other folks have tried to generate derivatives of the D-DIN fonts with intermediate weights, apparently using automated tools. I did not think those extra weights look good, so I did not adopt that approach.

For print work, I'm using DINish Regular in black and white and find the result to be eminently readable, but your use case may differ from mine. The [fonts.playbeing.com](https://fonts.playbeing.com/dinish) website is built using the Slate theme, which I did not touch except of course replacing the font.

In short, I'm not sure what the point is that you're trying to make, but I hope I have adressed your concern. If not, please elucidate.

**Comment by alarcondgtuc** [comment]
Oh yeah, sorry for the lack of details.

It was on the latest version of Firefox, on Windows 10. Here I'm comparing it to Hind, which is a typeface I specifically chose because of its heavier regular weight compared to others. Mainly to replace fonts on every website because It's easier to read for me.

As about DINish, I like it! it's an awesome work, although I personally don't see it as a body text font I'm sure it has many other good uses. I apologize if I sounded rude.

On a side note, I'm curious, is there a roadmap for whats to come for DINish?

**Comment by driehuis** [comment]
No offense taken!

There is no formal roadmap for DINish. I took up DINish because there was no freely usable DIN font on any of the major CDNs, in the hope that Google would pick it up once the quality was to their standards. It is a project I'm doing in my spare time, and I cannot commit to anything except the basic upkeep. Making Google's quality control tools happy was a major undertaking, and as the tools progress and add requirements, keeping them happy is a task in itself.

When I started what I thought was a process just to control the quality of DINish, I had no clue I'd be doubling its glyph count! Neither had I imagined what a source of joy this would be.

I do have plans and ambitions. I'd love to add the full 100-900 weights. I'd love to do a multi-master version, may even a variable font. Adding Vietnamese, Cyrillic and Greek is on the wish list. Full Opentype `frac` support would be cool.

If you look under the hood, you'll find that the bits that DINish were made of have organically grown. Many were the times I thought I broke something, only to find that the inconsistencies were already in the original D-DIN release. I'm trying to straighten them out as I continue, to make the wish list items feasible. In particular, the `features.fea` files need work, and are on the critical path to add new weights. It works for the current weights so it hasn't bubbled to the top of my priorities.

There are too many bad fonts out there, and I do not want to add any half-baked features to DINish.

So there's the wish list. It would not be truthful to call it a roadmap, because I can't promise any of it will happen.

Seeing that you're interested in legibility, let me share my surprise finding. I added old style numerals to DINish just because I could. I thought it would just be a gimmick. One day, I was working on a spreadsheet. I think I was using DejaVu Sans, because it has tabular numbers, but anyway, I was tired and I felt my eyes glazing over. As a distraction, I selected DINish with its freshly added tabular old style numerals, and was surprised to find that the old style numerals were much easier to read when tired. Switching back to the previous font I immediately started making reading errors again. Maybe it's just me, but I switched my spreadsheet's font to DINish tabular old style and never looked back.

Anyway, closing this issue as there is no feasible fix in the foreseeable future for your use case.
